docs/design/communication_library.md

```diff
@@ -0,0 +1,23 @@
+## Communication Library
+
+We believe there is no need to re-implement a communication library compatible with data structures differing from this dl framework to another. Therefore, FTLib will import a user-specific communication library during initialization. 
```

Comment:
data structures differing from this dl framework to another -> maybe "framework-specific data structures"